searx/engines/sepiasearch.py

```diff
@@ -0,0 +1,82 @@
+# SepiaSearch (Videos)
+#
+# @website     https://sepiasearch.org
+# @provide-api https://framagit.org/framasoft/peertube/search-index/-/tree/master/server/controllers/api
+# @using-api   yes
+# @results     JSON
+# @stable      yes
+# @parse       url, title, content, publishedDate, thumbnail
+
+from json import loads
+from dateutil import parser, relativedelta
+from urllib.parse import urlencode
+from datetime import datetime
+
+categories = ['videos']
+paging = True
+language_support = True
+time_range_support = True
+safesearch = True
+supported_languages = [
```

Comment:
Can you add in a comment the source of this list ?

Reply:
Which source? For `supported_languages`?

Reply:
The source for `supported_languages` is https://framagit.org/framasoft/peertube/search-index/-/blob/master/client/src/views/Search.vue#L618-641

Reply:
I added the URL in the commit message:  https://github.com/searx/searx/commit/53c8d945b489b1185d89e02bbc6b6ea6b60a91da